addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="blind1">
+        <label>RFXCOM Blind1 thing</label>
+        <description></description>
```

Comment:
This is tricky, because I don't have idea where this different things (messages) are really used. I only have few RF devices my self, so most probably community support is needed here to provide correct description (etc).